### PR TITLE
YAML - SV germline

### PR DESCRIPTION
**Pull request by dmaziec**

### CODE REVIEW

File: portal_objects/workflows/20_unrelated_SV_filter_vcf-check.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+## Workflow information #####################################
+#     General information for the workflow
+#############################################################
+name: 20_unrelated_SV_filter_vcf-check
+description: Filter out common and artefactual SV calls through comparison to 20 unrelated files. |
```
Comment (dmaziec) [inline portal_objects/workflows/20_unrelated_SV_filter_vcf-check.yaml]:
```suggestion
description: Filter out common and artifactual SV calls through comparison to 20 unrelated files. |
```

File: portal_objects/workflows/BICseq2_map_norm_seg.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    argument_type: file.tar_gz
+
+  chrlist:
+    argument_type: file.txt
+
+  fasta:
+    argument_type: file.fa
+
+## Output information #######################################
+#     Output files and quality controls
```
Comment (dmaziec) [inline portal_objects/workflows/BICseq2_map_norm_seg.yaml:40]: there are no quality controls, is it still required to put this in the comment?

File: portal_objects/workflows/granite-filter_SV_selector_vcf-check.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  # File argument
+  input_vcf:
+    argument_type: file.vcf_gz
+
+  genes:
+    argument_type: file.txt
+
+  # Parameter argument
+  SV_types:
+    argument_type: parameter.json
```
Comment (dmaziec) [inline portal_objects/workflows/granite-filter_SV_selector_vcf-check.yaml:37]: what about the other input parameters? 

```
      {
          "argument_type": "parameter",
          "workflow_argument_name": "VEP"
      },
      {
          "argument_type": "parameter",
          "workflow_argument_name": "VEPsep"
      },
      {
          "argument_type": "parameter",
          "workflow_argument_name": "VEPtag"
      },
      {
          "argument_type": "parameter",
          "workflow_argument_name": "VEPrescue"
      },
      {
          "argument_type": "parameter",
          "workflow_argument_name": "aftag"
      },
      {
          "argument_type": "parameter",
          "workflow_argument_name": "afthr"
      },
```

File: portal_objects/metaworkflows/CNV_germline.yaml [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+
+    ## Specific arguments ##############
+    ####################################
+    input:
+
+      # File argument
+      input_vcf:
+        argument_type: file.vcf_gz
+        source: SV_length_filter_vcf-check
+        source_argument_name: length_filtered_SV_vcf
```
Comment (dmaziec) [inline portal_objects/metaworkflows/CNV_germline.yaml:470]: missing: 
```
        {
          "argument_name": "max_length",
          "argument_type": "parameter"
        }
```

File: portal_objects/workflows/BICseq2_add_confidence.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+#     Input files and parameters
+#############################################################
+input:
+
+  # File argument
+  input_vcf:
+    argument_type: file.vcf_gz
+
+## Output information #######################################
+#     Output files and quality controls
```
Comment (dmaziec) [inline portal_objects/workflows/BICseq2_add_confidence.yaml:24]: there are no quality controls, is it still required to put this in the comment?

File: portal_objects/workflows/BICseq2_map_norm_seg.yaml (hunk 2 above)
Comment (B3rse) [reply portal_objects/workflows/BICseq2_map_norm_seg.yaml:40]: I kinda used the same comment structure for all the yaml and would leave it like that

File: portal_objects/workflows/granite-filter_SV_selector_vcf-check.yaml (hunk 3 above)
Comment (B3rse) [reply portal_objects/workflows/granite-filter_SV_selector_vcf-check.yaml:37]: there is a default on the cwl and we were not setting a different one in the metaworkflows, https://github.com/dbmi-bgm/cgap-pipeline-SV-germline/blob/main/portal_objects/metaworkflows/cnv_germline.json or https://github.com/dbmi-bgm/cgap-pipeline-SV-germline/blob/main/portal_objects/metaworkflows/SV_germline.json

File: portal_objects/metaworkflows/CNV_germline.yaml (hunk 4 above)
Comment (B3rse) [reply portal_objects/metaworkflows/CNV_germline.yaml:470]: should be there I think https://github.com/dbmi-bgm/cgap-pipeline-SV-germline/blob/62f9a0a4eea1553ad932d9ccd18044adec5dbfa9/portal_objects/metaworkflows/SV_germline.yaml#L75 

can you double check?

File: portal_objects/workflows/BICseq2_add_confidence.yaml (hunk 5 above)
Comment (B3rse) [reply portal_objects/workflows/BICseq2_add_confidence.yaml:24]: same as above

**Pull request merged**